src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -272,7 +272,10 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 				if (imageCalloutResult.embedImage) {
 					let base64String = await this.getFileContentBase64(URI.file(imageCalloutResult.imagePath));
 					let mimeType = await this.getFileMimeType(URI.file(imageCalloutResult.imagePath));
-					this.cellModel.addAttachment(mimeType, base64String, path.basename(imageCalloutResult.imagePath).replace(' ', ''));
+					let imageName = this.cellModel.addAttachment(mimeType, base64String, path.basename(imageCalloutResult.imagePath).replace(' ', ''));
```

Comment:
Image names with multiple spaces won't render correctly since replace only replaces the first space here.